packages/oidc-middleware/README.md

```diff
@@ -283,10 +295,10 @@ const oidc = new ExpressOIDC({ /* options */ });
 app.use(oidc.router);
 
 function addUserContext(req, res, next) {
-  if (!req.userinfo) return next();
+  if (!req.userContext.userinfo) return next();
```

Comment:
Shouldn't this be `if (!req.userContext)` ?  Also, let's put the return statement on a newline with brackets